src/sparsezoo/model/model.py

```diff
@@ -531,14 +562,14 @@ def _directory_from_files(
         display_name: Optional[str] = None,
         regex: Optional[bool] = False,
         allow_multiple_outputs: Optional[bool] = False,
-        **kwargs,
+        **kwargs: object,
     ) -> Union[Directory, None]:
 
         # Takes a list of file dictionaries and returns
         # a Directory() object, if successful,
         # otherwise None.
         if all([file_dict.get("file_type") for file_dict in files]):
-            # if file_dict is retrieved from the API as `request_json`
+            # if file_dict is retrieved from the API as `files`
```

Comment:
I would not change this comment. mentioning `request_json` clearly indicates that the source was the remote server, while `files` is very generic and may be confusing (e.g. for new people who join neural magic and get to work with this codebase)